src/typing_manager.ts

```diff
@@ -160,28 +159,27 @@ export class TypingManager implements Disposable {
         await this.client.input(`<c-o>${keys}`);
     };
 
-    private handleCompositeEscapeFirstKey = async (key: string): Promise<void> => {
+    // whenever j or k is pressed,
+    //   if last key pressed was the other (j for k, k for j)
+    //      and pressed recently enough,
+    //   then backspace and simulate esc
+    private handleCompositeEscape = async (key: string): Promise<void> => {
         const now = new Date().getTime();
-        if (this.compositeEscapeFirstPressTimestamp && now - this.compositeEscapeFirstPressTimestamp <= 200) {
-            // jj
+        if (this.compositeEscapeFirstPressKey &&
+            this.compositeEscapeFirstPressKey === (key === 'j' ? 'k' : 'j') &&
```

Comment:
This line doesn't seem to do anything and I can't figure out why -- I only wanted `jk` and `kj`, not `jj` and `kk` too. nbd for me but surely strange.